### PR TITLE
Update community-treasury.md

### PR DESCRIPTION
**Pull request by Phyloiota**
Updated to current state of the projects

# Description of change

Please write a summary of your changes and why you made them.

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes #issue_number`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix
- Documentation Enhancement

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [ ] I have performed a self-review of my own changes
- [ ] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: common/community/funding/community-treasury.md [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
 description: A Decentralized Community governed Fund to support projects in the IOTA Ecosystem and Support the community
 ---
 
-# Community Treasury
+# IOTA Community Treasury
 
 ![Community Treasury](https://github.com/iota-community/Community-Governance/raw/main/assets/iota_community_treasury.png)
 
-The community treasury is envisioned to be a project where the community can apply for funding and the community can also decide if they want to approve the funding.
+The IOTA community treasury will be set up as soon as the IOTA network gets upgraded with the Stardust ledger, and smart contracts can be executed on the Mainnet. The IOTA Community will be able to use a budget of 50 TI to support the ecosystem via grants. Untill this is possible, the community is also deciding on setting up a [Community Treasury Grant Committee](https://govern.iota.org/t/sgp-0005-shimmer-community-treasury-grant-committee/1576) for the Shimmer network.
```
Comment (Dr-Electron) [inline common/community/funding/community-treasury.md:10]:
```suggestion
The IOTA community treasury will be set up as soon as the IOTA network gets upgraded with the Stardust ledger, and smart contracts can be executed on the Mainnet. The IOTA Community will be able to use a budget of 50 TI to support the ecosystem via grants. Until this is possible, the community is also deciding on setting up a [Community Treasury Grant Committee](https://govern.iota.org/t/sgp-0005-shimmer-community-treasury-grant-committee/1576) for the Shimmer network.
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

File: common/community/funding/community-treasury.md [hunk 2]
```diff
@@ -1,22 +1,15 @@
...
 
 ## Initial Voting
 
-First there will be an initial voting on if the community wants to have a community treasury or if they want to burn the unclaimed tokens. You can read more about the unclaimed tokens in [this](https://blog.iota.org/unclaimed-tokens-community-treasury-and-iota-as-faq/) blog post from the IF.
+The IOTA community has decided to set up the IOTA Community Treasury in IOTA's first-ever Governance vote completed on 14 June 2022. In the following [Blog - article](https://blog.iota.org/community-treasury-vote-final-results/), you will be able to see the vote results and find all further information regarding the initial vote to set up the IOTA Treasury.
```
Comment (Dr-Electron) [inline common/community/funding/community-treasury.md:15]:
```suggestion
The IOTA community has decided to set up the IOTA Community Treasury in IOTA's first-ever Governance vote completed on 14 June 2022. In the following [Blog - article](https://blog.iota.org/community-treasury-vote-final-results/), you will be able to see the vote results and find all further information regarding the initial vote to set up the IOTA Treasury.
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

**Review by Dr-Electron — APPROVED** [review]
LGTM

**Review by Phyloiota — COMMENTED** [review]
Feinie

**Pull request merged**